src/app/shared/mocks/angulartics.service.mock.ts

```diff
@@ -1,4 +1,7 @@
+import { Injectable } from '@angular/core';
+
 /* tslint:disable:no-empty */
+@Injectable({providedIn: 'root'})
```

Comment:
It seems unlikely that a mock service would need providedIn: root

Reply:
done